### PR TITLE
Vmo 4205 community landing page with language adder

### PR DESCRIPTION
**Pull request by jcbashdown**
This includes the language adder changes and so we are targeting the 3917 PR. Once this is QA'd it can be merged down without interfering with QA on 3917.

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -52,18 +102,32 @@ import {forEach, isEmpty} from 'lodash'
...
   {
     mixins: [lang, Routes],
+    components: {LanguageAdder},
   },
 )
 class Home extends Vue {
   @Prop({default: () => ({})}) readonly appConfig!: object
   @Prop({default: () => ({})}) readonly builderConfig!: object
 
+  extraContentHidden: boolean = true
```
Comment (safydy-r) [inline src/views/Home.vue:119]: Can we pls add `boolean` prefix as usually ? (is, should, ...)

File: src/components/interaction-designer/flow-editors/import/LanguageAdder.vue [hunk 2]
```diff
@@ -148,6 +149,7 @@ class LanguageAdder extends mixins(Lang, Routes) {
       bcp_47: '',
     }
     this.selected_iso_639_3 = {}
+    this.selected_iso_3166_1 = {}
```
[the remark this answers is not in the document]
Comment (jcbashdown) [reply src/components/interaction-designer/flow-editors/import/LanguageAdder.vue:152]: You can see how it's used in the bcp 47 here:

https://www.rfc-editor.org/rfc/bcp/bcp47.txt

See section 2.1

[the remark this answers is not in the document]
Comment (safydy-r) [reply src/components/interaction-designer/flow-editors/import/LanguageAdder.vue:152]: thanks

**Comment by adeablay** [comment]
@jcbashdown 
Based on [the design](https://www.figma.com/proto/ebuqgagNxDLn3dBIbSiNmA/DSN-166:-[Community-Flow-Builder]-Landing-Page-for-Creating-Importing-Flows?node-id=2034%3A865&scaling=min-zoom&page-id=0%3A1&show-proto-sidebar=1&starting-point-node-id=2031%3A840) we have,
- **On the landing page**, the Create Flow button background colour is should be same colour as the "Import/paste a json file" text colour.
- The Add language button text and border colours should be the same as above. 

**Create Flow Page**
- We have a different layout from what is in the Design.
- Buttons below the Create Flow form should be two i.e Cancel and Create Flow buttons instead of one, Save and Continue button. Please see images below.

**Import Json page**
- The "Import File" button background colour should be same as Create Flow button.

**Comment by jcbashdown** [comment]
> @jcbashdown
> Based on [the design](https://www.figma.com/proto/ebuqgagNxDLn3dBIbSiNmA/DSN-166:-%5BCommunity-Flow-Builder%5D-Landing-Page-for-Creating-Importing-Flows?node-id=2034%3A865&scaling=min-zoom&page-id=0%3A1&show-proto-sidebar=1&starting-point-node-id=2031%3A840) we have,
> 
> * **On the landing page**, the Create Flow button background colour is should be same colour as the "Import/paste a json file" text colour.
> * The Add language button text and border colours should be the same as above.
> 
> **Create Flow Page**
> 
> * We have a different layout from what is in the Design.
> * Buttons below the Create Flow form should be two i.e Cancel and Create Flow buttons instead of one, Save and Continue button. Please see images below.
> 
> **Import Json page**
> 
> * The "Import File" button background colour should be same as Create Flow button.

On the first two I think you are right but I don't think that changes are expected for the Import and Create flow pages here - I think Amen is just reusing the old designs from before those pages were implemented? My understanding is we are only making changes to the landing page. I might fix Import File/Create Flow style mismatch though. Need to look into why that's happening. I'll also fix those first two.

**Pull request merged**